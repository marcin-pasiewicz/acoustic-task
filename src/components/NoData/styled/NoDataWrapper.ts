import styled from 'styled-components'

export const NoDataWrapper = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-items: center;
  align-items: center;
  padding: 25px;
`;
