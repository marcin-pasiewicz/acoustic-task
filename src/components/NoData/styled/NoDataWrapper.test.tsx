import React from 'react';
import { render } from '@testing-library/react';
import { NoDataWrapper } from './NoDataWrapper';

describe('<NoDataWrapper/>', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<NoDataWrapper/>);
    expect(asFragment()).toMatchSnapshot();
  });
});
