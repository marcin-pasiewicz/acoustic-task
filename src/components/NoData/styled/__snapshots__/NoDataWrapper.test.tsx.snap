// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoDataWrapper/> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-AxjAm hvoVVA"
  />
</DocumentFragment>
`;
